src/Symfony/Component/Finder/Comparator/NumberComparator.php

```diff
@@ -73,7 +73,6 @@ public function __construct(?string $test)
             }
         }
 
-        $this->setTarget($target);
-        $this->setOperator($matches[1] ?? '==');
+        parent::__construct($target, $matches[1] ?: '==');
```

Comment:
did you change ?? to ?: on purpose?

Reply:
Yes. `setOperator()` accepted an empty string and I've made the constructor stricter in that regard.